### ISSUE TITLE
Add passport.js support

### ISSUE THREAD

**Issue by elleryfamilia** [petecoop/generator-express#216]
These would be nice
- [ ] Add option to include passport.js
- [ ] Add option for selecting passport strategies to be supported

**Comment by petecoop** [comment]
Yeah would be nice, I've actually mentioned this on #78 but not really had the time to work on those
